scripts/releases/upload-release-assets-for-dotslash.js

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
the logic in this file is quite complex. It would be nice to add some unit tests to make sue it behaves as we expect.